lib/re_glob.mli

```diff
@@ -26,7 +26,7 @@ exception Parse_error
 
 val glob :
   ?anchored:bool ->
-  ?period:bool ->
```

Comment:
Just to confirm, this doesn't break compatibility because we haven't released this with `?period` yet?